<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctor_sign_up</class>
 <widget class="QDialog" name="doctor_sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="first_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="last_name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="email"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="password"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="phone_number"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>National ID</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="national_id"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QGroupBox" name="gender_groub_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="male">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="female">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Specialization</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="specialization"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="price"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="validation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>559</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>validation</string>
   </property>
  </widget>
  <widget class="QWidget" name="days" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
